platform/src/main/java/platform/pageobjects/Footer/FooterPage_PT90_106.java

```diff
@@ -0,0 +1,723 @@
+package platform.pageobjects.Footer;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import base.NewBaseClass;
+import platform.pageobjects.Authentication.LoginPage;
+import utils.CommonFunctions;
+import utils.XLHandler;
+
+/**
+ * @Author : Chetan Sonparote
+ * @Date of Creation : 26 June 2021
+ * @Description: Additional Locator variables and Methods for footer.
+ */
+
+public class FooterPage_PT90_106 {
+	List<String[]> testdata;
+	public WebDriver driver;
+	private static Logger log = LogManager.getLogger(FooterPage_PT90_106.class.getName());
+
+	String[] shipmentdata;
+	int waitTime = 5;
+
+	public FooterPage_PT90_106(WebDriver driver) {
+		this.driver = driver;
+		PageFactory.initElements(driver, this);
+	}
+
+	@FindBy(xpath = "//div[@class='footer-bt']//li[1]//a[1]")
```

Comment:
Plz replace abs xpath with relative ones

Reply:
I checked the xpaths for Footer links but they have same class so It when I added those classes xpaths, it was not identifying. So had to take unique class from from webpage and had to add xpath from then on.

Reply:
In case of design change, all these links will not be found as it is absolute path. Plz replace to relative xpath

Reply:
@csonparote san, I have updated the xpaths.

Reply:
xpath not updated in this file